ecs-cli/modules/clients/aws/amimetadata/client.go

```diff
@@ -85,3 +106,29 @@ func (c *metadataClient) GetRecommendedECSLinuxAMI(architecture string) (*AMIMet
 	err = json.Unmarshal([]byte(aws.StringValue(response.Parameter.Value)), metadata)
 	return metadata, err
 }
+
+func isARM64Instance(instanceType string) bool {
```

Comment:
I think this function and `architectureFor` could just be one function

Reply:
Good point, done!